tvplayer/src/main/java/com/kaltura/tvplayer/offline/exo/ExoOfflineManager.java

```diff
@@ -471,7 +477,7 @@ public void prepareAsset(@NonNull PKMediaEntry mediaEntry, @NonNull SelectionPre
         }
 
         MediaItem mediaItem = builder.build();
-        if (mediaFormat != PKMediaFormat.dash) {
+        if (mediaFormat != PKMediaFormat.dash || mediaFormat != PKMediaFormat.hls) {
```

Comment:
@GouravSna  sounds like it has to be && and not ||

Reply:
Done